File: server/models/nation_rb.js
const mongoose = require("mongoose");
const NationSchemaRb = new mongoose.Schema({
  date: Date,
  rb_battles_mean: Number,
  rb_air_frags_per_battle: Number,
  nation: String,
  rb_battles_sum: Number,
  rb_win_rate: Number,
  rb_air_frags_per_death: Number,
  rb_ground_frags_per_battle: Number,
  rb_ground_frags_per_death: Number,
  rb_lower_br: Number,
  cls: String,
});

const rb_ranks_0 = mongoose.model("rb_ranks_0", NationSchemaRb, "rb_ranks_0");
const rb_ranks_1 = mongoose.model("rb_ranks_1", NationSchemaRb, "rb_ranks_1");
const rb_ranks_all = mongoose.model("rb_ranks_all", NationSchemaRb, "rb_ranks_all");
module.exports = { rb_ranks_0, rb_ranks_1, rb_ranks_all };

File: server/src/main/java/com/example/server/nationrb/NationRb.java
package com.example.server.nationrb;

import org.springframework.data.mongodb.core.mapping.Field;

import com.example.server.Nation;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public abstract class NationRb extends Nation {

    @Field("rb_lower_br")
    private Float lowerBr;
    @Field("rb_win_rate")
    private Float winRate;
    @Field("rb_battles_sum")
    private Float battleSum;
    @Field("rb_battles_mean")
    private Float battleMean;

    @Field("rb_ground_frags_per_battle")
    private Float groundFragPerBattle;
    @Field("rb_ground_frags_per_death")
    private Float groundFragPerDeath;

    @Field("rb_air_frags_per_battle")
    private Float airFragPerBattle;
    @Field("rb_air_frags_per_death")
    private Float airFragPerDeath;
}


